pkgs/development/libraries/stargate-libcds/Makefile.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/Makefile b/Makefile.b
+index 872af46..7eba8a1 100644
+--- a/Makefile
++++ b/Makefile.b
```

Comment:
```suggestion
+++ b/Makefile
```

Reply:
@SuperSandro2000 did you mean
```diff
--- a/Makefile
+++ b/Makefile
```
?

Reply:
yeah, GitHub just can't render that with a suggestion.